.github/workflows/build.yml

```diff
@@ -10,15 +10,24 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Need to check out if free tier on github can run macos and windows jobs. Can you confirm they can?

Reply:
I think they can! I have been using it since 2019? 2020? (Sorry, don't remember the exact year) Example repo below, see https://github.com/emacs-sideline/sideline.